src/Components/ArtistName.jsx

```diff
@@ -0,0 +1,23 @@
+//The AlbumName component formats Album names, adding a delimiter between them when there are multiple artists for a single album.
+
+//The AlbumName component takes three props: name (artist's name), index (position in the list), and totalArtists (total number of artists).
+//It uses an inner function called generateDelimiter to decide whether to add a delimiter(" & ") between artist names based on the index and totalArtists.
+const ArtistName = ({ name, index, totalArtists }) => {
+    const generateDelimiter = (index, length) => {
+        if (index < length - 1) {
+            return " & "
+        }
+        return ""
+    };
```

Comment:
This looks like a good way to solve this issue! I didn't figure it out last week and need to try this.